backend/test_suite_generator.py

```
@@ -15,6 +15,7 @@
 from datetime import datetime, timezone, timedelta
 from typing import Dict, Any, Optional
 import logging 
+import jwt # Added jwt import
```

Ruff (0.12.2)

`jwt` imported but unused

Remove unused import: `jwt`

(F401)